BikeData/src/main/scala/by/itechart/Main.scala

```diff
@@ -0,0 +1,7 @@
+package by.itechart
+
+object Main extends App {
+  ParsingLogic.generalStats()
+  ParsingLogic.usageStats()
```

Comment:
If Main class calls ParsingLogic 3 times, it means it know too much about that class
Either ParsingLogic does 3 non-related things and needs to be split into pieces or ParsingLogic is actually having only 1 methods which for some reason is split into 3 different APIs